libs/designsystem/src/lib/scss/base/_lists.scss

```diff
@@ -0,0 +1,89 @@
+$bullet-font-size: 1.3em;
+$hollow-bullet-font-size: 0.8em;
+$list-item-line-height: 1.2em;
+
+// TODO: Move to utilities/functions
+@function unitless($number) {
+  @return $number / ($number * 0 + 1);
+}
+
+// -----------------------------------------------------------------------------
+// Grid layout
+// -----------------------------------------------------------------------------
+li {
+  display: grid;
+  grid-template-columns: auto 1fr;
+
+  // Nested lists
+  ol,
+  ul {
+    grid-column: 2 / -1;
+  }
+}
+
+// -----------------------------------------------------------------------------
+// Spacing
+// -----------------------------------------------------------------------------
+ol,
+ul {
+  padding-inline-start: size('xxxs'); // left
+}
+
+li {
+  &::before {
+    padding-inline-end: size('xs'); // right
+  }
+
+  // Vertical space between list items (excluding the first outermost item)
+  li,
+  & + li {
+    margin-block-start: size('xxs'); // top
+  }
```

Comment:
Reverting this declaration should probably also be part of the opt-out guidelines.